internal/impl/amqp1/config.go

```diff
@@ -28,6 +28,7 @@ const (
 	targetAddrField  = "target_address"
 	appPropsMapField = "application_properties_map"
 	metaFilterField  = "metadata"
+	persistentField  = "persistent"
```

Comment:
The naming of the field could be confusing in the context of the AMQP 1.0 standard. The AMQP 0.9 standard distinguishes between "durable" and "persistent."  These are different terms with slightly different meaning. 

- Durable: A server resource that survives a server restart.
- Persistent: A message that the server holds on reliable disk storage and MUST NOT lose after a server restart.

The AMQP 1.0 standard only "knows" durability (durable). My conclusion would be: it should be rather named "durable"

https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-header

https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-header